src/utils.jl

```diff
@@ -36,6 +36,19 @@ function sideof(p::Point{2,T}, s::Segment{2,T}) where {T}
   ifelse(area > atol(T), :LEFT, ifelse(area < -atol(T), :RIGHT, :ON))
 end
 
+"""
+    sideof(point, line)
+
+Determines on which side of the oriented `line`
+the `point` lies. Possible results are `:LEFT`,
+`:RIGHT` or `:ON` the line.
+"""
+function sideof(p::Point{2,T}, l::Line{2,T}) where {T}
+  a, b = l.a, l.b
+  area = signarea(p, a, b)
+  ifelse(area > atol(T), :LEFT, ifelse(area < -atol(T), :RIGHT, :ON))
+end
+
```

Comment:
we actually need to remove the method with segment as it implements the case with line. please start a separate PR where the method is fixed to work with line, and fix all uses accordingly.